### ISSUE TITLE
How to add own HTML and specify setAutoHideAnimation property

### ISSUE THREAD

**Issue by aslamj** [alexbeletsky/ng-notifications-bar#50]
I'm doing:
    notificationsConfigProvider.setAutoHideAnimation('fadeOutNotifications');
    notificationsConfigProvider.setAutoHideAnimationDelay(15 * 1000);	
But I don;t see any animations at hide time when I'm running the example.

Also is there a way to specify own HTML for message? When notificationsConfigProvider.setAcceptHTML(true), can I put in any HTML or it has limited capabilities.

**Comment by PhilipTodd** [comment]
Yes, it's not clear how the setAutoHideAnimation() method should be used. It'd be nice if someone could give a quick explanation or example. 

Does the 'fadeOutNotifications' parameter refer to the css class?

**Comment by jcubic** [comment]
:+1:
